app/auth/middleware.test.js

```diff
@@ -15,12 +15,6 @@ const userLocationFailureStub = {
   },
 }
 
-function UserStub({ name, roles = [], locations = [] } = {}) {
```

Comment:
I wonder if we do want to do this. For me this ties these two unit tests together and makes this test care about how the User library is structured.

Maybe this middleware test should just be checking that the User class is called with the correct arguments that are built during this middleware?